juju-charms/charm-trilio-data-mover-api/src/lib/charm/openstack/dmapi.py

```diff
@@ -0,0 +1,140 @@
+import os
+import charmhelpers.contrib.openstack.utils as ch_utils
+import charms_openstack.charm
+import charms_openstack.adapters
+import charms_openstack.ip as os_ip
+
+DMAPI_DIR = '/etc/dmapi'
+DMAPI_CONF = os.path.join(DMAPI_DIR, 'dmapi.conf')
+
+
+class DmapiAdapters(charms_openstack.adapters.OpenStackAPIRelationAdapters):
+    """
+    Adapters class for the Data Mover API charm.
+    """
+
+    def __init__(self, relations):
+        print(relations)
+        super(DmapiAdapters, self).__init__(
+            relations,
+            options_instance=charms_openstack.adapters.APIConfigurationAdapter(
+                port_map=DmapiCharm.api_ports))
+
+
+class DmapiCharm(charms_openstack.charm.HAOpenStackCharm):
+
+    # Internal name of charm + keystone endpoint
+    service_name = name = 'dmapi'
+
+    # First release supported
+    release = 'queens'
+
+    # Packages the service needs installed
+    packages = ['python-nova']
+
+    # Init services the charm manages
+    services = ['trilio-data-mover-api']
```

Comment:
can we keep this name consistent. I saw @maheshbhoge using as 'tvault-datamover-api'. Please check with him